articles/virtual-desktop/create-host-pools-user-profile.md

```diff
@@ -63,4 +66,4 @@ To configure the virtual machines with the FSLogix software, do the following on
 | VHDLocations        | Multi-String Value | "Network path for file share"     |
 
 >[!IMPORTANT]
->To help secure your Windows Virtual Desktop environment in Azure, we recommend you don't open inbound port 3389 on your VMs. Windows Virtual Desktop doesn't require an open inbound port 3389 for users to access the host pool's VMs. If you must open port 3389 for troubleshooting purposes, we recommend you use [just-in-time VM access](https://docs.microsoft.com/azure/security-center/security-center-just-in-time).
\ No newline at end of file
+>To help secure your Windows Virtual Desktop environment in Azure, we recommend you don't open inbound port 3389 on your VMs. Windows Virtual Desktop doesn't require an open inbound port 3389 for users to access the host pool's VMs. If you must open port 3389 for troubleshooting purposes, we recommend you use [just-in-time VM access](https://docs.microsoft.com/azure/security-center/security-center-just-in-time).
```

Comment:
Make the link into a relative link since it's in the same repo: (../security-center/security-center-just-in-time.md).

This way the link will stay the same even if the URL changes.